chainer_chemistry/functions/activation/softmax.py

```diff
@@ -0,0 +1,12 @@
+from chainer import functions
+
+
+def softmax(x, axis=1, mask=None, mask_value=1e10):
+    if mask is None:
+        h = x
+    else:
+        if x.shape != mask.shape:
```

Comment:
We can check the range of value. (0. <= x <= 1.)
But we can add later.

Reply:
let us add later.